libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -42,6 +42,7 @@ class AP_RCProtocol {
         ST24,
         FPORT,
         FPORT2,
+        SBUS_200kbaud,
```

Comment:
Such an ugly name! No consistency with the other naming conventions. FAST_SBUS? DJI_SBUS?

Reply:
I was trying not to tie it to a specific product.

No argument on "ugly", 'though :-)

Reply:
We're going to go with FastSBUS, it seems - way less ugly :-)

Reply:
Fixed the butt-ugliness issue.